### ISSUE TITLE
Error while selecting date

### ISSUE THREAD

**Issue by smrkarkii** [princid/Expense-Tracker#4]
Dates from future can also be selected while adding income/expense. I guess, it should be disabled.

**Comment by princid** [comment]
Yeah! Thanks for addressing this issue, it should be disabled. Feel free to make a pull request if you can fix it!

**Comment by smrkarkii** [comment]
Thanks, I'll try.

**Comment by Im-Kunal-13** [comment]
@smrkarkii I would like to work on this. Can you assign this to me?

**Comment by princid** [comment]
@anil-muraleedharan Yeah, this issue is still on. You can work on it.

**Comment by anil-muraleedharan** [comment]
@princid I have raised a [PR](https://github.com/princid/Expense-Tracker/pull/38/) for this issue could you review it
